src/types.ts

```diff
@@ -3,7 +3,7 @@ import { InternalSchema } from '.';
 
 export type StandardJSONSchema = JSONSchema4;
 
-export type MongoDBJSONSchema = Pick<StandardJSONSchema, 'title' | 'required' | 'description'> & {
+export type MongoDBJSONSchema = Partial<StandardJSONSchema> & Pick<StandardJSONSchema, 'title' | 'required' | 'description'> & {
```

Comment:
Still allowing all JSON Schema fields so that things like JSON Schema's `type` can be used.